examples/gno.land/r/gnoland/valopers/v2/valopers.gno

```diff
@@ -6,10 +6,11 @@ import (
 	"std"
 
 	"gno.land/p/demo/avl"
+	"gno.land/p/demo/dao"
 	"gno.land/p/demo/ufmt"
 	pVals "gno.land/p/sys/validators"
-	govdao "gno.land/r/gov/dao"
-	"gno.land/r/sys/validators"
+	govdao "gno.land/r/gov/dao/v2"
```

Comment:
should not depend on a specific govdao version.

either you use another mechanism; or you can create a `r/valopers/v2/govdaov2` helper package, so that we can create just the missing bridge if we switch to govdao v3.

Reply:
I've tried to apply the proxy pattern you've mentioned, and I think it fits in nice. Realms that require an expanded GovDAO API will just define their own "bridge", or have some kind of update mechanism.

Let me know what you think:

[d98d780](https://github.com/gnolang/gno/pull/2581/commits/d98d78051c2c91044d0c3d87e83f9b08e5441d68)